nero_view.py

```diff
@@ -0,0 +1,211 @@
+import customtkinter
```

Comment:
We should rename this to main_view as this will be the initial screen, maybe home_view.

Reply:
my thinking was that `nero` conveys that part tbh

Reply:
what do you think about making a folder for nero_view called pages, and then another folder for mockmodel and raspberrymodel called models

Reply:
Going off of my controller comment, I think Nero terminology should be reserved for classes/objects that control switching between views and other large scale things rather than just another name for main